modules/vulnerability-management20-retention-settings.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-retention-settings_{context}"]
+= Configuring vulnerability management report retention settings
+
+[role="_abstract"]
+You can configure settings that determine when vulnerability report requests expire and other retention settings for reports.
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Platform Configuration* -> *System Configuration*. You can configure the following settings for vulnerability reporting:
+* *Expired vulnerability requests*: The number of days after which a vulnerability report request expires. The default value is 90 days. ##Is there a maximum and minimum value?##
+* *Vulnerability report run history retention*: The number of days that a record is kept of vulnerability reports that have been run. This setting controls how many days that reports are listed in the *All report jobs* tab under *Vulnerability Management 2.0* -> *Vulnerability Reporting* when a report is selected. ##Default, min, and max values?##
```

Comment:
There are some exceptions to the jobs that will be deleted from "All report jobs" by this retention setting. I think they can be mentioned here too.
Following jobs will not be removed by this setting:
1) Jobs in WAITING or PREPARING state.
2) Last successful scheduled report job.
3) Last successful On demand emailed report job.
4) Last successful downloadable report job.
5) Downloadable report jobs for which the report file has not been deleted (either manually or by Downloadable report pruning settings)

The reason for not deleting last successful jobs in exceptions 2), 3) and 4) is to support filters like "CVEs since last successful scheduled report" in the report configuration.